README.md

```diff
@@ -471,45 +459,46 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
 * When defining binary operators, name the parameter `other`(`<<` and `[]` are
   exceptions to the rule, since their semantics are different).
 
-* The names of predicate methods (methods that return a boolean value) should
-  end in a question mark (i.e. `Array#empty?`). Methods that don't return a
-  boolean, shouldn't end in a question mark.
+* Name predicate methods with a `?`. Predicate methods are methods that return a
+  boolean value.
```

Comment:
Imo it's also an imortant property that they don't have a side effect but iirc that's somewhat contentious

Reply:
Although I very much agree with you I think mentioning that would mean introducing new rules to the style guide, which I am avoiding in this PR.